src/plugins/conceptPlugin.js

```diff
@@ -67,18 +68,29 @@ export default function createConceptPlugin(options = {}) {
     );
   };
 
-  const embedToHTML = (embed, locale) => {
+  const embedToHTML = async (embed, locale) => {
     const {
       id,
       title: { title },
       content: { content },
-      metaImage,
     } = embed.concept;
 
+    const visualElement = defined(embed.concept.visualElement, {
+      visualElement: '',
+    });
     const copyright = defined(embed.concept.copyright, {});
     const authors = defined(copyright.creators, []).map(author => author.name);
     const license = defined(copyright.license, {}).license;
     const source = defined(embed.concept.source, '');
+
+    const transformed = await transform(
```

Comment:
Kaller transform inni anna transform, men så lenge forklaring ikkje har anna forklaring som visuelt element burde det ikkje være et problem. Sender også inn visuelt element som body og ikkje i visualElement-feltet som article bruker.

Reply:
Hvorfor er det et problem om det er en forklaring? Burde vi validere det i backend i så fall?

Reply:
Det er kanskje ikkje et problem. Eg tenkte berre at vi ikkje måtte få en uendelig rekursjon. Det er jo egentlig kun mogeleg dersom det er forklaringa sjølv som er visuelt element og det gir ikkje meining.

Reply:
Ah skjønner, ja det er jo en potensiell fare, selvom det gir fint lite mening å ha en forklaring med den samme forklaringen som visuelt element. 
Kunne kanskje lagt inn en sjekk på det?

Reply:
Det er ikkje mogeleg å gjøre via gui og eg ser ikkje for meg at nokon kjem til å gjøre det via api heller.